docs/userguide/expectations.md

```diff
@@ -916,13 +919,13 @@ To change the behavior of `NULL = NULL` comparison pass the `a_nulls_are_equal =
 
 This matcher supports only compound data-types comparison. It check if the actual set contains all values of expected subset.
 
-When comparing data using `contain` matcher, the data-types of columns for compared compound types must be exactly the same.
+When comparing data using the `contain` matcher, the data-types of columns for compared compound types must be exactly the same.
 
-The matcher supports all advanced comparison options as `equal` like: `include` , `exclude`, `join_by` etc..
+The matcher supports all advanced comparison options supported by `equal` like: `include` , `exclude`, `join_by` etc..
```

Comment:
It does not sounds good as word support is repeated twice in same sentence. Not sure if change is required here.

Reply:
I am not sure if the following construct is correct English: *'to support all Xs as Y'*. Maybe it's just my ignorance, but perhaps *The matcher supports the same advanced comparison options as `equal`* would be better?

For now, I reverted it to the original form.